web-wallet/package-lock.json

```diff
@@ -60,7 +59,7 @@
         "vitest-canvas-mock": "0.3.3"
       },
       "engines": {
-        "node": ">=18.0.0"
+        "node": ">=22.0.0"
```

Comment:
Is this upgrade expected? I guess it's needed for w3sper? If not, perhaps we should stick to the LTS version instead (v20.18.0)? 👀

Reply:
no: it should be `>=20`.

I forgot to re-generate the package-lock file.